Add config: 'no_data_partition' (#82)
harvester/docs: docs/install/harvester-configuration.md

  vip_mode: dhcp
  vip_hw_addr: 52:54:00:ec:0e:0b
```

### `install.no_data_partition`

#### Definition

> Do not create an additional disk partition for storing VM data
> and use operating system partition to store them. Default: `false`.

Reviewer: ```suggestion
Do not create an additional disk partition for storing VM data. The operating system partition will be used to store the VM data. 

Default: `false`.
```

Author: Rephrased it a bit based on your suggestion.

> and use operating system partition to store them. Default: `false`.

Reviewer 2: Do not create additional disk partitions for VM data, the system partition will be used by default.
Can't be resized or reverted after the installation, useful when users want to use additional disks for the VM data with [auto-provision-path]() config. Default: `false`.

Author: Rephrased it a bit based on your suggestion.
